cmds/dhclient/dhclient.go

```diff
@@ -153,6 +158,21 @@ func main() {
 		debug = log.Printf
 	}
 
+	// if we boot quickly enough, the random number generator
```

Comment:
I read into this problem once. It's possible to save the output of /dev/random to non-volatile storage and use it to partially seed the entropy on the next boot. There is an example of this in `man 4 random`. Make sure the stored seed is only accessible by root. It might not make it unblock any faster unless you use the `RNDADDENTROPY` ioctl to add to the entropy pool, but that might introduce rather bad security implications.